.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js: 6
+cache:
+  yarn: true
+  directories:
+    - node_modules
+script:
+  - yarn test
```

Comment:
`yarn` is executed by default. If you remove `test` from `scripts`, you have to remove it everywhere 😉